registry/quilt_server/models.py

```diff
@@ -244,3 +245,8 @@ class Comment(db.Model):
     contents = db.Column(db.String(MAX_COMMENT_LENGTH), nullable=False)
 
     package = db.relationship('Package', back_populates='comments')
+
+class Role(db.Model):
+    id = db.Column(postgresql.UUID, primary_key=True)
+    name = db.Column(db.String(200), unique=True)
+    arn = db.Column(db.String(200))
```

Comment:
Does it make sense to make ARN the primary key? It should at least be unique to prevent adding multiple entries in the registry db for the same IAM Role.

Reply:
I think it shouldn't be. If an admin creates a role and assigns a bunch of users to it and thne wants to change the ARN they should be able to with our changing the friendly_name of the role. Similarly, they may want to have the same ARN for different friendlynames